src/test/java/atdd/station/StationAcceptanceTest.java

```diff
@@ -24,7 +23,8 @@
 
     @Test
     public void createStation() {
-        String stationName = "강남역";
+        //커밋을 위한 주석 추가
+        String stationName = "강남역(2)";
```

Comment:
```suggestion
        String stationName = "강남역";
```

`(2)`는 노선을 의미하는 것으로 보이는데요.
지금처럼 지하철역 이름에 포함하여 서버에 전달할 경우, 서버에서 파싱해서 처리해야 할 텐데요. 이를 분리하는 것이 좋을거 같아요.

그리고 현재 인수조건에는 `지하철 역을 등록한다` 로, 이 부분은 요구사항에 해당되지 않네요 ㅎ;